languages/python/reference/concepts/argument_unpacking.md

```diff
@@ -0,0 +1,6 @@
+# Argument unpacking
+
+TODO: ADD MORE
+
+- the example solution for this exercise uses `splat` (`*`) to unpack rows for the `zip()` function. [matrix](../exercise-concepts/matrix.md)
```

Comment:
I never knew the names for the `splat` and `splatty-splat` operators before. I love Python.

Reply:
The first is in more common usage ... I'm not sure I'd put money on the second.

Reply:
Quick searching shows `double-splat` is probably more common.